### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by anandkumarcoder**
Feat: input and image

### CODE REVIEW

File: Styles/image.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.full-image{
+    object-fit: contain;
+     max-width: 100%;
+     max-height: 100%;
+  
+}
+
+.round-image{
+    /* object-fit: contain; */
```
Comment (ravikumar0403) [inline Styles/image.css:9]: you can delete this commented code if not required

File: Styles/input.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    position: absolute;
+    right: 5px;
+    bottom: 10px;
+  }
+  #username{
+    border: 2px solid #000000;  
+  }
+#username:invalid{
+    border: 2px solid #ff0000;
+}
```
Comment (ravikumar0403) [inline Styles/input.css:28]: use variables for colors

File: docs/docs.html [hunk 3]
```diff
@@ -632,91 +643,185 @@ <h4>Example of icon buttons</h4>
...
           </div>
 
-          <div class="card-container one">
-            <img
-              class="card-image"
-              src="/assets/tshirt-image.jpg"
-              alt="T-shirt image"
-            />
+          <div class="code-snippets">
+            <pre><code class="html">&lt;div class=&quot;input-holder&quot;&gt;
```
Comment (chanchal16) [inline docs/docs.html:690]: Indent it properly. To remove white spaces from the code block you can style the code tag either by giving white-space:pre-line/pre-wrap or by giving a suitable negative left margin.

File: Styles/input.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+#username:invalid + span::before{
+    content: '✖';
+  color: #ff0000;
```
Comment (chanchal16) [inline Styles/input.css:32]: use variables for colors

File: docs/docs.html (hunk 3 above)
Comment (anandkumarcoder) [reply docs/docs.html:690]: Will add this new patch

**Pull request merged**